File: final_dataset_A/generated_code/claude/python/load_application_settings_yaml-claude.py
import yaml

def load_settings(yaml_file):
    with open(yaml_file, 'r') as f:
        settings = yaml.load(f, Loader=yaml.FullLoader)
    return settings

if __name__ == '__main__':
    settings = load_settings('config.yaml')
    print(settings)

File: generated-code-groupA-updated/generated-code-groupA-updated/claude/java-claude-A/LoadApplicationSettingsYAML-claude.java
import org.yaml.snakeyaml.Yaml;
import java.io.FileInputStream;
import java.io.InputStream;
import java.util.Map;

public class LoadApplicationSettingsYAML {

    public static Map<String, Object> loadSettings(String filePath) throws Exception {
        Yaml yaml = new Yaml();
        try (InputStream input = new FileInputStream(filePath)) {
            return yaml.load(input);
        }
    }

    public static void main(String[] args) throws Exception {
        Map<String, Object> settings = loadSettings("config.yaml");
        System.out.println(settings);
    }
}
